cmd/osmosisd/main.go

```diff
@@ -13,7 +13,7 @@ import (
 func main() {
 	params.SetAddressPrefixes()
 	rootCmd, _ := cmd.NewRootCmd()
-	if err := svrcmd.Execute(rootCmd, osmosis.DefaultNodeHome); err != nil {
+	if err := svrcmd.Execute(rootCmd, "OSMOSISD", osmosis.DefaultNodeHome); err != nil {
```

Comment:
shouldn't this be lowercase?

Reply:
this created an env prefix, usually env vars are all caps (I think you might be thinking that this is referencing the cli cmd, which I believe is different)

Reply:
ah that is what I thought